### ISSUE TITLE
Load screen

### ISSUE THREAD

**Issue by hi-names-nat** [Geeknerd1337/NSCM#52]
the hangup time on load is quite frustrating. let's add a smooth load screen. something like the "Loading..." popup in hl

**Comment by Geeknerd1337** [comment]
I agree with this, @hi-names-nat would you mind making a basic UI mockup of this for me and I will attempt to implement it this week.
